mkt/comm/models.py

```diff
@@ -45,7 +45,8 @@ def check_acls(user, obj, acl_type):
     elif acl_type == 'admin':
         return acl.action_allowed_user(user, 'Admin', '%')
     elif acl_type == 'reviewer':
-        return acl.action_allowed_user(user, 'Apps', 'Review')
+        return (acl.action_allowed_user(user, 'Apps', 'Review') or
```

Comment:
do you care about checking the obj type so you don't give addon reviewers access to apps and vice versa?